sdk/core/azure-core/test/libcurl-stress-test/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
```

Comment:
Would it make sense to include a link to a definition of what a Helm Chart is? For those ignorant souls who might be reading it?

Reply:
I agree with @LarryOsterman, I am not familiar with helm charts at all. Package manager for kubernetes?

Can you also share some context, maybe in the PR description, what the purpose of this PR is? What are we trying to accomplish and why (what's motivating this)?

Reply:
This is the C++ SDK. Everyone working in the C++ SDK is expected to have a working knowledge of C++. 

However we're *NOT* expected to have working knowledge of helm charts, so something that helps us understand them would be greatly appreciated.

Reply:
updated the readme